controllers/managedocs_controller_test.go

```diff
@@ -757,15 +755,14 @@ var _ = Describe("ManagedOCS controller", func() {
 						Expect(sc.Spec.AllowRemoteStorageConsumers).Should(BeTrue())
 					}
 
-					// prometheus tls -- will be uncommented
-					// By("Creating a prometheus service resource")
-					// utils.WaitForResource(k8sClient, ctx, promServiceTemplate.DeepCopy(), timeout, interval)
-					// By("Creating serving cert CA bundle configmap")
-					// utils.WaitForResource(k8sClient, ctx, caBundleConfigMap.DeepCopy(), timeout, interval)
+					By("Creating a prometheus service resource")
+					utils.WaitForResource(k8sClient, ctx, promServiceTemplate.DeepCopy(), timeout, interval)
+					By("Creating serving cert CA bundle configmap")
+					utils.WaitForResource(k8sClient, ctx, caBundleConfigMap.DeepCopy(), timeout, interval)
 					By("Creating a prometheus resource")
 					utils.WaitForResource(k8sClient, ctx, promTemplate.DeepCopy(), timeout, interval)
-					// prometheus tls -- will be uncommented
-					// assertPrometheusTLSConfigValues(ctx, k8sClient)
+
+					assertPrometheusTLSConfigValues(ctx, k8sClient)
```

Comment:
No need for a function here. please assert what is needed to be asserted in the body of the test case.

Reply:
I haven't worked on test yet so please don't review unit test.